### ISSUE TITLE
Why expose the `softmax` and `sigmoid` parameters on the `DiceFocalLoss` if by default the `FocalLoss` works only with logits?

### ISSUE THREAD

**Issue by ramonemiliani93** [Project-MONAI/MONAI#2610]
**Please use MONAI's Discussions tab**
Hi, I find it confusing that the `softmax` and `sigmoid` parameters are exposed on the `DiceFocalLoss` if by default it has to work with the logits, as per the documentation:

```
input: the shape should be BNH[WD]. The input should be the original logits
                due to the restriction of ``monai.losses.FocalLoss``.
```

Is there a specific reason for not removing the parameters? I feel this may lead to bugs on user code.
